Resources/Locale/en-US/deltav/store/uplink-catalog.ftl

```diff
@@ -1,2 +1,25 @@
+# Weapons
+
+# Explosives
+
+# Ammo
+
+# Utility
+uplink-reinforcement-radio-nukie-mouse-name = Mouse Operative Reinforcement Teleporter
+uplink-reinforcement-radio-nukie-mouse-desc = Calls in a specially trained mouse operative to assist you.
+
+# Implants
+
+# Bundles
+
+# Tools
+
+# Job Specific
+
+# Armor
+
+# Misc
+
+# Pointless
 uplink-bionic-syrinx-implanter-name = Bionic Syrinx Implanter
 uplink-bionic-syrinx-implanter-desc = An implant that enhances a harpy's natural talent for mimicry to let you adjust your voice to whoever you can think of.
```

Comment:
These should be moved to another comment category

Reply:
Where are they supposed to go?

Reply:
Oh wait, implants. Alright.

Reply:
Fixed!